config.yml

```diff
@@ -6218,17 +6540,6 @@ production_recipes:
       Iron Door:
         material: IRON_DOOR
         amount: 18
-  Forge_Iron_Trapdoors:

```

Comment:
This removal is still here; still a regression. Iron Trapdoors are live on the server now, I doubt removing them is intentional.